### ISSUE TITLE
Can't use cyrilic letters for Nickname

### ISSUE THREAD

**Issue by cradleguard** [GuruRAM/NBullet-Hell#1]
Cyrilic letters cannot be used for Nickname

**Comment by GuruRAM** [comment]
Yes, I will change the validation since cyrilic letters are supported by https://fonts.google.com/specimen/Press+Start+2P :)
